modules/actions/github.go

```diff
@@ -25,17 +25,48 @@ const (
 	githubEventPullRequestComment       = "pull_request_comment"
 )
 
-func convertFromGithubEvent(evt *jobparser.Event) string {
+// canGithubEventMatch check if the input Github event can match any Gitea event.
+func canGithubEventMatch(evt *jobparser.Event, triggedEvent webhook_module.HookEventType) bool {
```

Comment:
Is it possible to redefine it like:
```suggestion
func canGithubEventMatch(eventName string, triggedEvent webhook_module.HookEventType) bool {
```

And could you please add some unit tests for it?

Reply:
Of course. I will add some tests.